src/scenes/home/informationForm/informationForm.js

```diff
@@ -16,6 +16,7 @@ class SignupInformation extends Component {
     super(props);
     this.onIdentifierStatusChange = this.onIdentifierStatusChange.bind(this);
     this.onCheckBoxChange = this.onCheckBoxChange.bind(this);
+    this.previousPage = this.previousPage.bind(this);
```

Comment:
Is there any particular reason you are binding previous page?

Reply:
I'd be willing to make this PR as-is if you were to file a low-priority ticket to make the Interests resetting section more resilient to changes after its merged. Let me know!

Reply:
@Cooperbuilt How urgent is the current issue? I can try making the resetting more resilient and add it to this pull request

Reply:
```
let step = this.step;
getStep() {
  return ++step;
}
```